README.md

```diff
@@ -1,2 +1,62 @@
 # comp110-worksheet-B
```

Comment:
Flowchart is OK in terms of control flow, though try to use the standard flowchart node shapes discussed in class for decision nodes etc.

The presentation of your flowchart is untidy, which makes it a little hard to follow. Most graphics software has snap-to-grid options which will help you keep things tidy, and some software has tools specifically for drawing flowcharts.